internal/bminventory/inventory.go

```diff
@@ -431,10 +430,7 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 		ignitionParams["ServiceIPs"] = dataurl.EncodeBytes([]byte(ignition.GetServiceIPHostnames(b.Config.ServiceIPs)))
 	}
 
-	if cluster.ImageInfo.StaticIpsConfig != "" && params.ImageCreateParams.ImageType == models.ImageTypeFullIso {
-		ignitionParams["StaticIPsData"] = base64.StdEncoding.EncodeToString([]byte(cluster.ImageInfo.StaticIpsConfig))
-		ignitionParams["StaticIPsConfigScript"] = base64.StdEncoding.EncodeToString([]byte(constants.ConfigStaticIpsScript))
-	}
+	// [TODO] - add static network configuration only in case it was provided and this is a FullIso image
```

Comment:
Will it be done in a different PR?

Reply:
yes, all part of https://issues.redhat.com/browse/MGMT-3867